core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -237,6 +239,7 @@ object KafkaConfig {
   val NumNetworkThreadsProp = "num.network.threads"
   val NumIoThreadsProp = "num.io.threads"
   val BackgroundThreadsProp = "background.threads"
+  val NumReplicaAlterLogDirsThreadsProp = "num.replica.alter.log.dirs.threads"
```

Comment:
Though the request is named "AlterReplicaLogDirsRequest", the thread is `ReplicaAlterLogDirsThread`, which is more consistent with `ReplicaFetcherThread`. Because this config is used to determine the thread number, it seems more intuitive to name it `num.replica.alter.log.dirs.threads`. What do you think?